std/conv.d

```diff
@@ -3934,7 +3934,6 @@ private T octal(T)(const string num)
     return value;
 }
 
-///
 @system unittest
```

Comment:
This is a private method - there's no point in exposing in trying to expose the `unittest`